frontend/src/components/landing/PlanMatrix.module.scss

```diff
@@ -0,0 +1,365 @@
+      }
+
+      .pricing-overview {
+        display: flex;
+        flex-direction: column;
+        gap: $spacing-md;
+
+        .price {
+          @include text-body-lg;
+          color: $color-blue-50;
+          font-weight: 600;
+        }
+
+        .pick-button {
+          border: 2px solid $color-blue-50;
+          border-radius: $border-radius-sm;
+          padding: $spacing-sm;
+          text-align: center;
+          color: $color-blue-50;
+          font-weight: 700;
+
+          &:hover {
+            background-color: $color-blue-50;
+            color: $color-white;
+            cursor: pointer;
+          }
+        }
+
+        small {
+          text-align: center;
+        }
+      }
+    }
+  }
+}
+
+.mobile {
+  padding: $spacing-lg;
+
+  .plans {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $spacing-lg;
+    list-style-type: none;
+
+    .plan {
+      flex: 1 0 $content-xs;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: $spacing-lg;
+      background-color: $color-white;
+      border-radius: $border-radius-md;
+      padding: $spacing-md;
+
+      h3 {
+        @include text-title-2xs;
+        font-weight: 600;
+        text-align: center;
+      }
+
+      &.recommended h3 {
+        // .plan adds a padding of $spacing-md on all sides of the h3.
+        // Since we want the h3 to spand the full width of the plan, and to be
+        // flush with the top of the .plan, we enlarge it by that amount on the
+        // sides and at the top:
+        width: calc(100% + 2 * $spacing-md);
+        margin-top: -1 * $spacing-md;
+        margin-inline: -1 * $spacing-md;
+        padding: $spacing-md;
+        background-color: $color-violet-80;
+        color: $color-white;
+        border-radius: $spacing-sm;
+      }
+
+      .feature-list {
+        flex: 1 0 auto;
+        display: flex;
+        flex-direction: column;
+        gap: $spacing-sm;
+        width: 100%;
+        list-style-type: none;
+
+        li {
+          display: flex;
+
+          .description {
+            flex: 1 0 auto;
+
+            svg {
+              display: inline;
+              vertical-align: middle;
+            }
+          }
+          .availability {
+            flex: 0 1 $layout-lg;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            svg {
+              color: $color-blue-60;
+            }
+          }
+        }
+      }
+
+      .pricing {
+        display: flex;
+        flex-direction: column;
+        gap: $spacing-md;
+        width: 100%;
+
+        .discount-notice {
+          @include text-body-md;
+          font-weight: 700;
+          text-align: center;
+        }
+
+        .pricing-overview {
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          gap: $spacing-md;
+          width: 100%;
+          $billingMonthlyNoteHeight: 18px;
```

Comment:
question (non-blocking): is `spacing-md` (16px) not enough here?

Reply:
In terms of heigh, it probably would be! But that feels more like using it because the current value of that variable happens to be (roughly) what I need, not because I actually need "a medium amount of spacing" here (what I actually need is "roughly the height of a sentence in a somewhat smaller font", but I'm not sure if there's a value that represents that?).